cli/pkg/console/console.go

```diff
@@ -59,7 +59,8 @@ func Log(format string, args ...interface{}) {
 }
 
 func Debug(format string, args ...interface{}) {
-	l.Printf(format+"\n", args...)
+	blue := color.New(color.FgBlue).SprintfFunc()
+	l.Printf(blue("\n"+format, args...))
```

Comment:
I experimented with updating Error and Info colors here just to get some Go-fu. We don't have to stick with these actual color values in the long-run, but they appear to be working fine. 👍 Go is fun.